governing-documents/security.md

```diff
@@ -0,0 +1,266 @@
+---
+layout: default
+title:  Security Policy
+parent: Governing Documents
+grand_parent: Hyperledger TOC
+nav_order: 10
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+This is a template for a security vulnerability disclosure policy for
+Hyperledger projects or subprojects within individual Hyperledger projects
+(we note that some projects may have different repositories with different
+security policies).  Please edit this as appropriate for your project and
+delete this top section.  For most projects, this will only involve minor
+edits (look for text in red below).  For projects that have security experts,
```

Comment:
There is no red text in this document. Do we need to add formatting to allow for the red text to show? Or is there another way to represent this text in markdown?

Reply:
It looks like colored text is complicated in markdown, unfortunately.